### PR TITLE
i18n(zh-cn): Update troubleshooting.mdx

### PR DESCRIPTION
**Pull request by liruifengv**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- Update troubleshooting.mdx

#### Related Issue / Implementation PR

- https://github.com/withastro/docs/pull/4831

### CODE REVIEW

File: src/content/docs/zh-cn/guides/troubleshooting.mdx [hunk 1]
```diff
@@ -6,9 +6,62 @@ i18nReady: true
...
+<Debug {{answer: sum(2, 4)}} />
+<Debug {answer} />
+```
+
 ## 常见错误信息
 
-下面是你在终端可能会看到的一些常见错误信息，包括它们可能的含义及解决方案。
+下面是你在终端可能会看到的一些常见错误信息，包括它们可能的含义及解决方案。查阅我们的 [完整错误参考指南](/zh-cn/reference/error-reference/)，了解你可能遇到的所有 Astro 错误。
 
 ### Cannot use import statement outside a module（不能在模块外使用导入语句）
```
Comment (100gle) [inline src/content/docs/zh-cn/guides/troubleshooting.mdx:66]:
```suggestion
### 不能在模块外使用导入语句
```

**Review by 100gle — CHANGES_REQUESTED** [review]
英文 title 可以去掉了

**Review by 100gle — APPROVED** [review]
LGTM

File: src/content/docs/zh-cn/guides/troubleshooting.mdx [hunk 2]
```diff
@@ -6,9 +6,62 @@ i18nReady: true
 
 Astro 提供了几个不同的工具来帮助你排除故障和调试你的代码。
 
+## 技巧和窍门
+
+### 用 `console.log()` 调试
+
+`console.log()`是一种简单但普遍的调试 Astro 代码的方法。你在哪里写`console.log()`语句将决定你的调试输出被打印在哪里。
```
Comment (Genteure) [inline src/content/docs/zh-cn/guides/troubleshooting.mdx:13]:
```suggestion
`console.log()` 是一种简单但普遍的调试 Astro 代码的方法。你写 `console.log()` 语句的位置将决定你的调试输出被打印在哪里。
```

File: src/content/docs/zh-cn/guides/troubleshooting.mdx [hunk 3]
```diff
@@ -6,9 +6,62 @@ i18nReady: true
...
+console.log('Hi! 我是服务器。这是在运行 Astro 的终端中记录的。');
+---
+<script>
+console.log('Hi! 我是客户端。这是在浏览器的开发控制台中记录的');
+</script>
+```
+
+Astro frontmatter 中的 `console.log()` 语句将始终输出到运行 Astro CLI 的 **终端**。这是因为 Astro 在服务器上运行，而不是在浏览器中。
+
+在 Astro `<script>` 标签中编写或导入的代码在浏览器中运行。任何 `console.log()` 语句或其他调试输出都将打印到 **浏览器中的控制台**。
```
Comment (Genteure) [inline src/content/docs/zh-cn/guides/troubleshooting.mdx:26]:
```suggestion
Astro frontmatter 中的 `console.log()` 语句将始终输出到运行 Astro CLI 的**终端**。这是因为 Astro 在服务器上运行，而不是在浏览器中。

在 Astro `<script>` 标签中编写或导入的代码在浏览器中运行。任何 `console.log()` 语句或其他调试输出都将打印到**浏览器中的控制台**。
```

Formatting, spaces are not necessary here.

File: src/content/docs/zh-cn/guides/troubleshooting.mdx [hunk 4]
```diff
@@ -6,9 +6,62 @@ i18nReady: true
...
+
+### 调试框架组件
+
+[框架组件](/zh-cn/core-concepts/framework-components/)（如 React 和 Svelte）是独特的。它们默认在服务器端渲染，这意味着 `console.log()` 的调试输出将在终端中可见。然而，在浏览器中它们也可能会被激活，这可能导致你的调试日志也出现在浏览器中。
```
Comment (Genteure) [inline src/content/docs/zh-cn/guides/troubleshooting.mdx:30]:
```suggestion
[框架组件](/zh-cn/core-concepts/framework-components/)（如 React 和 Svelte）较为特殊。它们默认在服务器端渲染，这意味着 `console.log()` 的调试输出将显示到终端中。然而，它们也可能会在浏览器中被激活，导致你的调试日志也出现在浏览器中。
```

Some rewording and reordering.

File: src/content/docs/zh-cn/guides/troubleshooting.mdx [hunk 5]
```diff
@@ -6,9 +6,62 @@ i18nReady: true
...
+
+这对于调试 SSR 输出和浏览器中的激活组件之间的差异非常有用。
+
+### Astro `<Debug />` 组件
+
+为了帮助你调试 Astro 组件，Astro 提供了内置 [`<Debug />`](/zh-cn/reference/api-reference/#debug-) 组件，你可以用它直接组件 HTML 模板中渲染任何值。这对于在浏览器中进行快速调试是很有用的，而不必在终端和浏览器之间来回切换。
```
Comment (Genteure) [inline src/content/docs/zh-cn/guides/troubleshooting.mdx:36]:
```suggestion
为了帮助你调试 Astro 组件，Astro 提供了内置的 [`<Debug />`](/zh-cn/reference/api-reference/#debug-) 组件，你可以用它把任意值直接渲染到你的 HTML 模板中。这对于在浏览器中进行快速调试是很有用的，而不必在终端和浏览器之间来回切换。
```

Rewording.

File: src/content/docs/zh-cn/guides/troubleshooting.mdx [hunk 6]
```diff
@@ -6,9 +6,62 @@ i18nReady: true
...
+---
+import { Debug } from 'astro:components';
+const sum = (a, b) => a + b;
+---
+
+<!-- 示例：在浏览器中输出 {answer: 6} -->
+<Debug answer={sum(2, 4)} />
+```
+
+调试组件支持多种语法选项，使调试更加灵活和简明。
```
Comment (Genteure) [inline src/content/docs/zh-cn/guides/troubleshooting.mdx:48]:
```markdown
The Debug component supports a variety of syntax options for even more flexible and concise debugging:
```

"The Debug component" 中的 "Debug" 特指 `<Debug />` 组件，所以不翻译比较好。
"Options" 表达多个可选的项目，这句话中文里不明确说“选项”就能表达出来同样的意思。

```suggestion
`Debug` 组件支持多种语法，使调试更加灵活和简明。
```

**Review by Genteure — APPROVED** [review]
LGTM

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🙌

**Pull request merged**